modules/rte_mis_mail/rte_mis_mail.module

```diff
@@ -381,3 +382,138 @@ function rte_mis_mail_mail($key, &$message, $params) {
       break;
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function rte_mis_mail_form_user_pass_alter(&$form, FormStateInterface $form_state, string $form_id) {
+
+  $title = $form['name']['#title'];
+  $markup = $form['mail']['#markup'];
+
+  if ($title instanceof TranslatableMarkup && $markup instanceof TranslatableMarkup) {
+
+    // Modify the string (e.g., add a prefix).
+    $modifiedTitle = t('Email address / Mobile number');
+
+    // Modify the string (e.g., add a prefix).
+    $modifiedString = t('Please use +91 for Mobile Number.<br>Password reset instructions will be sent to your registered email address and mobile number.');
+
+    // Replace the original markup with the modified one.
+    $form['mail']['#markup'] = $modifiedString;
+    // Replace the original markup with the modified one.
+    $form['name']['#title'] = $modifiedTitle;
+  }
+
+  // Setting the submit button disabled.
+  unset($form['#validate']);
+  unset($form['#submit']);
+  $form['#validate'][] = "rte_mis_mail_resend_custom_validate_handler";
```

Comment:
Why we are unsetting the existing validate functions. Can't we add multiple validation functions?